Sources/StreamVideo/WebRTC/StreamVideoCaptureHandler.swift

```diff
@@ -33,26 +35,28 @@ class StreamVideoCaptureHandler: NSObject, RTCVideoCapturerDelegate {
     }
     
     func capturer(_ capturer: RTCVideoCapturer, didCapture frame: RTCVideoFrame) {
-        Task { [weak self] in
-            guard let self else { return }
+        Task { [serialActor] in
+            await serialActor.enqueue { [weak self] in
```

Comment:
Not 100% sure why enqueuing async tasks helps performance, can you please elaborate one more time? 🙏

Reply:
The idea is that Tasks not running on a specific actor will run as part of the GlobalActor. The previous days I remember reading that the GlobalActor is concurrent. That meant that the Tasks created here (outside of an Actor) will run as part of the GlobalActor. As they will run concurrently we may doing processing for more that one frame at the same time which can be consuming.

However, looking again now I couldn't find what I was reading and i ended up asking ChatGPT. The answer is that the GlobalActor is serial. So I will remove this one.